.github/workflows/actions/docker-images-build/action.yml

```diff
@@ -0,0 +1,22 @@
+name: Action > Docker Images Build
+inputs:
+  image-build-tag:
+    required: true
+runs:
+  using: 'composite'
+  steps:
+    - uses: ./.github/workflows/actions/cached-dependencies
+    - uses: ./.github/workflows/actions/load-conf-env
+    - name: Build Docker Images
+      run: |
+        set -eo pipefail
+        source "$HOME/.nvm/nvm.sh"
+        source "$HOME/.sdkman/bin/sdkman-init.sh"
+        source $GITHUB_WORKSPACE/devops/linux/docker/conf.env
+        source $GITHUB_WORKSPACE/devops/linux/docker/conf/images/conf-app-images.sh 
```

Comment:
It looks like we will use the default app images for docker hub. I don't think these will be suitable as there are already images names that we should reuse - https://hub.docker.com/search?q=jempi

Reply:
Hmm...okay, noted. Will update as appropiate 👍🏽